build-package.sh

```diff
@@ -577,6 +607,12 @@ for ((i=0; i<${#PACKAGE_LIST[@]}; i++)); do
 		else
 			termux_error_exit "Unknown packaging format '$TERMUX_PACKAGE_FORMAT'."
 		fi
+		# saving a list of compiled packages for further work with it
+		termux_add_package_to_built_packages_list "$TERMUX_PKG_NAME"
```

Comment:
Add after `termux_step_finish_build`. Also, first letter of comments must be uppercase everywhere.

Reply:
> Add after `termux_step_finish_build`.

If you do so, the packages will not be added to the list. Since the `termux_step_finish_build` function exits the subshell, preventing it from doing so.

Reply:
Ah, right, missed that.